node-operators/block-producers/run-a-block-producer/block-producer-systemd.md

```diff
@@ -0,0 +1,170 @@
+
+Download the specification file executing:
+
+```bash
+wget https://raw.githubusercontent.com/papermoonio/external-files/main/Tanssi/Dancebox/dancebox-raw-specs.json
+```
+
+## Download the Relay Chain Specs File {: #download-relay-specs }
+
+The node binary file includes also the necessary code to run a relay chain node. When launching your Block Producer's node, it will also be required to provide the relay chain's specification file as a parameter. 
+
+Download the relay chain specification file executing:
+
+```bash
+wget https://raw.githubusercontent.com/papermoonio/external-files/main/Moonbeam/Moonbase-Alpha/westend-alphanet-raw-specs.json
+```
+
+## Setup the Systemd Service {: #setup-systemd-service }
+
+[Systemd](https://systemd.io/){target=\_blank} is a management system for Linux systems that manages services (daemons in Unix-like systems jargon), starting them automatically when the computer starts or reboots, or restarting them upon unexpected failures.
+
+The following commands configure a new account, the directory, and move the previously downloaded files to the right location.
+
+Create a new account to run the service:
+
+```bash
+adduser sequencer_service --system --no-create-home
+```
+
+Create a directory to store the required files and data:
+
+```bash
+mkdir /var/lib/sequencer-data
+```
+
+Set the folder's ownership to the account that will run the service to ensure writing permission:
+
+```bash
+sudo chown -R sequencer_service /var/lib/sequencer-data
+```
+
+And finally, move the binary and the chain specs to the folder:
+
+```bash
+mv ./tanssi-node /var/lib/sequencer-data && \
+mv ./westend-alphanet-raw-specs.json /var/lib/sequencer-data && \
+mv ./dancebox-raw-specs.json /var/lib/sequencer-data
+```
+
+### Create the Systemd Service configuration file {: #create-systemd-configuration }
+
+The next step is to create the Systemd configuration file. 
+
+You can create the file by running the following command:
+
+```bash
+sudo touch /etc/systemd/system/sequencer.service
+```
+
+Now you can open the file using your favorite text editor (vim, emacs, nano, etc) and add the configuration for the service:
+
+```bash
+[Unit]
+Description="Sequencer systemd service"
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=10
+User=sequencer_service
+SyslogIdentifier=sequencer
+SyslogFacility=local7
+KillSignal=SIGHUP
+ExecStart=/var/lib/sequencer-data/tanssi-node \
+--chain=/var/lib/sequencer-data/dancebox-raw-specs.json \
+--rpc-port=9944 \
+--name=sequencer-name \
+--base-path=/var/lib/sequencer-data/para \
+--state-pruning=2000 \
+--blocks-pruning=2000 \
+--collator \
+-- \
+--rpc-port=9946 \
+--name=sequencer-name-container \
```

Comment:
```suggestion
--name=tanssi-appchain \
```

Reply:
Normally we let them set a name that they want. If I'm running a collator under the brand "Tacos&Pizzas" I would like to put `tacospizzas-appchain`

Reply:
This applies to all the name tags. The benefit of having different names is for telemetry. So revisit this in all --name tags for collators and RPCs